docs/spark-qualification-tool.md

```diff
@@ -718,7 +719,7 @@ Sample output in text:
 +============+==============+==========+==========+=============+===========+===========+===========+====================+=======================================================+
 | appName-01 | app-ID-01-01 |    898429|    879422|       879422|  273911.92|       3.27|  624517.06|Strongly Recommended|                               |                       |
```

Comment:
Need to add the column headers

Reply:
Column headers are added above: `Unsupported Execs` and `Unsupported Expressions`